### PR TITLE
How to connect to Ruby console for direct Panoptes querying

### PR DESCRIPTION
**Pull request by alexbfree**
Added some documentation of the approach you pointed me to this morning @camallen . Thanks again.

### CODE REVIEW

File: science/panoptes_ruby_command_line.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+How to query Panoptes via Ruby console
+======================================
+
+Sometimes you will want to get live information from the Panoptes backend that is not available from the APIs. For staff and those with the appropriate level of access, the following approach is possible:
```
Comment (camallen) [inline science/panoptes_ruby_command_line.rst:4]: Looks good, can you please also add a note that this is super user access (with no constraints) and that people should take great care running heavy queries / doing data manipulations.

**Comment by alexbfree** [comment]
OK have added @camallen

**Comment by camallen** [comment]
@alexbfree i just realised this is in developers...instead should it be in https://github.com/zooniverse/how-to-zooniverse/tree/master/Devops ?

the content LGTM, just not sure about putting it in this repo

**Comment by alexbfree** [comment]
I suggested in https://github.com/zooniverse/rfc/issues/3 that we should merge the how-to-zooniverse and the developer.zooniverse.org repositories, not sure we really need two... for now i think it'd be fine to merge it here. We already have the page that tells you how to use this technique to edit subject sets in this repo anyway, and I'd rather not spend time reformatting it from RST to MD....

**Comment by camallen** [comment]
LGTM

**Comment by alexbfree** [comment]
👍

**Pull request merged**